node/alphanet/src/lib.rs

```diff
@@ -235,16 +221,15 @@ mod tests {
             None,
             3034,
             vec![bob.node_info.clone()],
-            chain_spec,
+            chain_spec.clone(),
         );
 
+        let (mut beacon_block, mut shard_block, shard_extra) =
+            alice.client.prepare_block(1, ChainPayload::default());
+        beacon_block.signature.authority_mask = vec![true; chain_spec.initial_authorities.len()];
+        shard_block.signature.authority_mask = vec![true; chain_spec.initial_authorities.len()];
         let (mut beacon_block, mut shard_block) =
-            match alice.client.try_produce_block(1, ChainPayload::default()) {
-                BlockProductionResult::Success(beacon_block, shard_block) => {
-                    (beacon_block, shard_block)
-                }
-                _ => panic!("Should produce block"),
-            };
+            alice.client.try_import_produced(beacon_block, shard_block, shard_extra);
         // Sign by bob to make this blocks valid.
         beacon_block.add_signature(&beacon_block.sign(bob.signer()), 1);
         shard_block.add_signature(&shard_block.sign(bob.signer()), 1);
```

Comment:
unrelated to this change, but does adding bob's signature and alice.client.try_import_blocks actually do anything for the test?
Seems like this try_import_blocks returns BlockImportingResult::AlreadyImported (and bob just accepts the block without his signature)

Reply:
It actually does not, because we set `beacon_block.signature.authority_mask = vec![true; chain_spec.initial_authorities.len()];`

Reply:
Given this PR, that should be removed. That was a quick fix befoe we add actual sigs.

Reply:
oh, you removed it. But you should then add signature by alice here.

Reply:
but even before this change these three lines don't do anything, `alice.client.try_import_blocks(beacon_block, shard_block);` just returns BlockImportingResult::AlreadyImported